### ISSUE TITLE
PRISM Kyrgyzstan can't connect to external server periodically, causes app functionality issues

### ISSUE THREAD

**Issue by wadhwamatic** [WFP-VAM/prism-app#872]
@ericboucher and @HarryMytilinaios - the PRISM Kyrgyzstan deployment utilizes an external server which seems to have sporadic availability. This causes the app to sit in a pending state (spinner spinning) for a long time before eventually rendering the basemap but preventing layers from being activated. I thought this PR would address this but seems to not have worked for this scenario.

You can see this in https://github.com/WFP-VAM/prism-app/pull/871 and the related surge deployment here:
https://prism-kgz-demo.surge.sh/

<img width="1512" alt="Screenshot 2023-06-16 at 14 40 01" src="https://github.com/WFP-VAM/prism-app/assets/3343536/3babd029-1210-48f5-b254-455f85544c14">

<img width="1431" alt="Screenshot 2023-06-16 at 14 39 45" src="https://github.com/WFP-VAM/prism-app/assets/3343536/b482cb34-5c72-4ed0-86b3-b1bcf09955b9">

_Originally posted by @wadhwamatic in https://github.com/WFP-VAM/prism-app/issues/861#issuecomment-1594329168_

**Comment by HarryMytilinaios** [comment]
@wadhwamatic I just opened the url and the wms getCapabilities is responding with 200. The PR I did was indeed fixing the issue of app hanging. For the rest endpoints the time is 15 seconds, so we will have to wait for 15 seconds and for the statictics endpoints the wait is 60 seconds, so we will have to wait until 60 seconds before the timeout. I can't reproduce it in the surge deployment above. Is there anything I can do to reproduce it?
